config/locales/ja.yml

```diff
@@ -14,85 +14,62 @@
 ---
 ja:
   x_comments:
-    one: 1件のコメント
     other: '%{count}件のコメント'
   x_followers:
-    one: 1人のフォロワー
     other: '%{count}人のフォロワー'
-  x_identifications:
-    one: 1件の同定
-    other: '%{count}件の同定'
+  x_identifications: '{{PLURAL%{count}件の同定}}'
```

Comment:
Hi @kueda

I am a part of the Wikimedia Language team, and am currently working with @Nikerabbit on Translatewiki.net and the Translate extension.

The Japanese pluralization issues are caused due to improper translation which caused the exported translations to be in the incorrect format. If a plural message translation is performed correctly the exported data will be in the format that you mentioned.  

Currently if a translator inputs an incorrect format of translation, Translatewiki.net does not warn them about it. We can update the translation checker to do so. This would still not stop translators from saving invalid translations, but should reduce the number of incorrect translations, since translators will be seeing a warning. 

We're [working on improving](https://phabricator.wikimedia.org/T204568) the message checker framework on Translatewiki.net to restrict translators from saving messages based on the translation errors. 

You can see an example checker added [here](https://github.com/wikimedia/translatewiki/tree/master/groups/EtherpadLite). Let me know if you are interested in using this or the improved checker framework. I'll create a task on Phabricator and follow through with you.

Reply:
Well, good to know you're working on it. We've just been deleting these translations before integration.

Reply:
I've [created a task on our issue board](https://phabricator.wikimedia.org/T224025), and have added iNaturalist as a project for the initial deployment of the validator framework. 

We will contact you again once the patch is merged and the validators can be added.